Directory.Packages.props

```diff
@@ -5,44 +5,44 @@
   <ItemGroup>
     <PackageVersion Include="Autofac" Version="7.1.0" />
     <PackageVersion Include="Autofac.Extensions.DependencyInjection" Version="8.0.0" />
-    <PackageVersion Include="AutoFixture.NUnit3" Version="4.18.1" />

```

Comment:
This one is not used in the BlueprintTemplate and it has a dependency on Nunit < 4